src/stamp/config.yaml

```
@@ -1,9 +1,10 @@
 preprocessing:
   output_dir: "/path/to/save/files/to"
   wsi_dir: "/path/containing/whole/slide/images/to/extract/features/from"
-
   # Extractor to use for feature extractor.  Possible options are "ctranspath",
-  # "mahmood-uni" and "mahmood-conch"
+  # "mahmood-uni", "mahmood-conch", "chief_ctranspath", "mahmood-conchv1_5",
```

```suggestion
  # "mahmood-uni", "mahmood-conch", "chief_ctranspath", "mahmood-conch1_5",
```
Just used your branch for feat extraction with new models, one minor fix is needed here:)